src/Microsoft.DotNet.Build.Tasks/Targets/coreAssembly.rsp

```diff
@@ -0,0 +1,2 @@
+﻿# CSC requires the runtimemetadataversion flag when building a core assembly (containing System.Object)
+/runtimemetadataversion:v4.0.30319
```

Comment:
I diffed a diag log from 12 and 14 and these are really the only properties that seem usefully different.